esmvaltool/diag_scripts/rainfarm/main.r

```diff
@@ -0,0 +1,128 @@
+# #############################################################################
+# rainfarm.r
+# Authors:       E. Arnone (ISAC-CNR, Italy)
+#	         J. von Hardenberg (ISAC-CNR, Italy) 
+# #############################################################################
+# Description
+# ESMValTool diagnostic calling the RainFARM library written in Julia (by von Hardenberg, ISAC-CNR, Italy).
+# RainFARM is a stochastic precipitation downscaling method, further adapted for climate downscaling.
+#  
+# Required
+# CDO
+# Julia language: https://julialang.org
+# RainFARM Julia library: https://github.com/jhardenberg/RainFARM.jl
```

Comment:
@mattiarighi @bouweandela : Is this a dependency that we allow? Julia

Reply:
I believe we were planning to support that? It is a popular open source scientific programming language.